### PR TITLE
[FrameworkBundle] Enable assets with templates only if the Asset component is installed

### PR DESCRIPTION
**Pull request by hason**
| Q             | A
| ------------- | ---
| Branch?       | 3.4
| Bug fix?      | no
| New feature?  | no
| BC breaks?    | no
| Deprecations? | no
| Tests pass?   | yes
| Fixed tickets | 
| License       | MIT
| Doc PR        | -

### CODE REVIEW

**Comment by nicolas-grekas** [comment]
could it make sense to merge this in 3.3?

File: src/Symfony/Bundle/FrameworkBundle/DependencyInjection/Configuration.php [hunk 1]
```diff
@@ -52,7 +52,7 @@ public function getConfigTreeBuilder()
 
         $rootNode
             ->beforeNormalization()
-                ->ifTrue(function ($v) { return !isset($v['assets']) && isset($v['templating']); })
+                ->ifTrue(function ($v) { return !isset($v['assets']) && isset($v['templating']) && class_exists(Package::class); })
```
Comment (Taluu) [inline src/Symfony/Bundle/FrameworkBundle/DependencyInjection/Configuration.php]: How about moving the class check at the start of the condition ? (Yes that's kind of a micro-optimization...)

Comment (nicolas-grekas) [reply src/Symfony/Bundle/FrameworkBundle/DependencyInjection/Configuration.php]: calling a function last *is* the best it terms of µoptim to me

**Review by nicolas-grekas — APPROVED** [review]

**Review by fabpot — APPROVED** [review]

**Comment by fabpot** [comment]
Thank you @hason.

**Pull request merged**